user/articles/120-search/11-mulitlanguage-search-sp1.markdown

```diff
@@ -0,0 +1,228 @@
+---
+header-id: multi-language-search-in-version-7-2-sp1
+---
+
+# Multi-Language Search in Version 7.2 SP1
+
+[TOC levels=1-4]
+
+How an asset's fields are indexed in the search engine plays an important role
+in the end user's experience. Not all assets are indexed in a way that supports
+searching in a language other than the default language. Even assets that are
+translatable might not support searching for the content in that language.
+
+Development surrounding the new (as of Liferay DXP Service Pack 1) search
+_synonyms_ feature precipitated improvements in how localized search
+works out-of-the-box. 
+
+In short, these assets contain text fields supporting localized search:
+
+| Asset | Fields | Localized Search Approach | Newly Supported in 7.2 SP1? |
+|-------|--------|---------------------------|-----------------------------|
+| Content Page | `title` | 2 | No |
+| Documents and Media Document | `content` | 3 | No |
+| Calendar | `name`, `description` | 1 | No |
```

Comment:
@brandizzi I just looked at this, and I do not think that Calendars are searchable via the Search widget, right?

How can you see, from the code, whether the asset is searchable via the search widget? Is there an easy place to see this?

Reply:
Is it  this call in the `*AssetRendererFactory` constructor?

```java
setSearchable(true);
```